lib/plug/templates/debugger.html.eex

```diff
@@ -167,7 +167,6 @@
 
     .exception-info > .detail {
         margin-top: 1.3em;
-        white-space: pre;
```

Comment:
We still want to preserve whitespace where we can though. Can you please try `pre-wrap` instead?

Reply:
`pre-wrap` looks good :+1: